src/propsToBody.js

```diff
@@ -47,13 +50,25 @@ function createShape(type, args) {
 }
 
 /**
- *
- * @param uuid {string}
- * @param props {BodyProps}
- * @param type {BodyShapeType}
- * @return {module:objects/Body.Body}
+ * @callback CreateMaterialCallback
```

Comment:
This doesn't look like a callback to me.

I'll go ahead and merge it and clean it up.

Reply:
Appreciate any help with JSDoc -- only familiar with TypeScript myself. I was trying to create a function type and use for this function and then again as an argument to another function.

Thanks for helping clean it up! I'm curious to see what the best way in JSDoc is and will keep in mind for next time.